tests/integration/kubernetes/lib.sh

```diff
@@ -372,3 +372,19 @@ get_node_kata_sandbox_id() {
 	done
 	echo $kata_sandbox_id
 }
+
+get_kata_sandbox_id_by_pod_name() {
```

Comment:
hmmm don't we have a function that return the sandbox id already?

Reply:
We do, it's right above this in the lib.sh, `get_node_kata_sandbox_id`. However, this line fails when I test it:
https://github.com/kata-containers/kata-containers/blob/46af7cf817b9efe2c8f3569b9341f305480c86fe/tests/integration/kubernetes/lib.sh#L362

"containerd-shim-kata-v2" doesn't exist on the process command line. I didn't want to remove this function, because I'm not sure if it was being retained for a different env (I did search and didn't see it being used anywhere). Instead, I added my new method which uses crictl to get the sandbox ID. The new method is also better because it allows you to filter on pod name in case multiple guests are running.